samples/doris-demo/udf-demo/src/main/java/org/apache/doris/udf/FunctionGrpc.java

```diff
@@ -0,0 +1,77 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.apache.doris.udf;
+
+import io.grpc.stub.StreamObserver;
+import net.devh.boot.grpc.server.service.GrpcService;
+import org.apache.doris.proto.FunctionService;
+import org.apache.doris.proto.PFunctionServiceGrpc;
+import org.apache.doris.proto.Types;
+
+import java.util.List;
+
+/**
+ * FunctionGrpc
+ *
+ * @author lirongqian
+ * @since 2022/02/08
+ */
+@GrpcService
+public class FunctionGrpc extends PFunctionServiceGrpc.PFunctionServiceImplBase {
+    
+    @Override
+    public void fnCall(FunctionService.PFunctionCallRequest request, StreamObserver<FunctionService.PFunctionCallResponse> responseObserver) {
+        String functionName = request.getFunctionName();
+        FunctionService.PFunctionCallResponse res;
+        if ("grpc_add".equals(functionName)) {
+            List<Types.PValues> argsList = request.getArgsList();
+            int sum = 0;
+            for (Types.PValues pValues : argsList) {
+                sum += pValues.getInt32Value(0) % 10;
```

Comment:
Why `% 10`?

Reply:
I was using it for testing, so I deleted it